frontend/dockerfile/docs/reference.md

```diff
@@ -1160,41 +1160,96 @@ The available `[OPTIONS]` are:
 - [`--link`](#add---link)
 - [`--exclude`](#add---exclude)
 
-The `ADD` instruction copies new files, directories or remote file URLs from `<src>`
-and adds them to the filesystem of the image at the path `<dest>`.
+The `ADD` instruction copies new files or directories from `<src>` and adds
```

Comment:
It would be nice to have some comparison of `ADD/COPY` or guidance on when to use which one. I've seen plenty of recommendations that "you should use COPY and not ADD," and while that might be a good idea for consistency, they sometimes imply that `COPY` is faster or smth. what is not true.

Reply:
We do have a section on this in the best practices document. https://docs.docker.com/build/building/best-practices/#add-or-copy

I can add a reference to that doc from these sections.

Reply:
We can of course expand on that section too - let me know if you have some suggestions